src/main/java/uk/gov/companieshouse/filinghistory/api/mapper/upsert/AbstractTransactionMapper.java

```diff
@@ -18,7 +18,7 @@ public abstract class AbstractTransactionMapper {
     protected static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyyMMddHHmmssSSSSSS")
             .withZone(UTC);
 
-    private final LinksMapper linksMapper;
+    protected final LinksMapper linksMapper;
```

Comment:
Can this be private?

Reply:
Yep, must have been a hangover from using the TopLevelTransactionMapper, done.